tests/test_models.py

```diff
@@ -8,3 +14,84 @@ def test_resnet50_rcnn():
     model = keras_rcnn.models.ResNet50RCNN(inputs, 21, 300)
 
     model.compile(loss=["mse", "mse", "mse"], optimizer="adam")
+
+
+def test_rpn():
+    options = {
+        "activation": "relu",
+        "kernel_size": (3, 3),
+        "padding": "same"
+    }
+
+    image = keras.layers.Input((224, 224, 3))
+
+    y_true = keras.layers.Input((None, 4), name="y_true")
+
+
+    features = keras.layers.Conv2D(64, **options)(image)
```

Comment:
I’d use a ResNet implementation from the keras-resnet package rather than VGG since we’ll consider this standard and match everything else.